nix/cardano-services/deployments/wallet-api.nix

```diff
@@ -0,0 +1,95 @@
+{
+  config,
+  values,
+  lib,
+  utils,
+  chart,
+  ...
+}: {
+  providers.wallet-api-provider = {
```

Comment:
We have no `wallet-api-provider` ready yet: please remove this

Reply:
This was part of my template testing, it has been removed